Website/Portals/_default/HotcakesViews/_default/Scripts/Checkout.js

```diff
@@ -914,12 +913,9 @@
                                 }
                             });
                     })
-                    .fail(function (xhr, status, error) {
-                        console.log("xhr: " + xhr);
-                        console.log("status: " + status);
+                    .fail(function (error) {
                         console.log("error: " + error);
```

Comment:
I know that this was already there before you began working on this, but we can't report an error directly to the console in production.  We need to instead display something more generic than the actual error message.  How much effort would it be to fix this in this PR?

Reply:
@VAlmea There hasn't been a response to this comment.  Please leave a comment telling me if this is resolved, or how else we might need to address this concern.